tests/unit/Makefile.am

```diff
@@ -321,8 +330,11 @@ connection_management_test_SOURCES = connection_management_test.c ../../cf-serve
 connection_management_test_LDADD = ../../libpromises/libpromises.la libtest.la ../../cf-serverd/libcf-serverd.la
 
 rlist_test_SOURCES = rlist_test.c \
-       ../../libpromises/rlist.c ../../libutils/logging.c
-rlist_test_LDADD = libtest.la libstr.la ../../libpromises/libpromises.la
+	../../libpromises/rlist.c \
+rlist_test_LDADD = libtest.la ../../libpromises/libpromises.la
```

Comment:
Oops, my bad, I left a stray ``\`` after the ``rlist.c`` entry there. It's easier if you fix it on your branch so I don't have to make a new PR.